### DISCUSSION TITLE
Shape of Buttons - New

### DISCUSSION THREAD

**Discussion by adrien-smz** [TomSchimansky/CustomTkinter#2018] [Q&A] [answered]
Hello,

I would like to have a button with a different shape. For example, a Star. So I want the button to be defined very precisely by the edges of the shape. 
I made some research and tried to upload an image with transparent background and make it a button.
But clicking far away outside from the edges  still makes it trigger the action as it considers a whole rectangle as button, despite the image having a transparent background outside of the star.

Is there any way to accomplish this in python ?
Thank you

**Comment by Akascape** [comment, marked as answer]
@adrien-smz Learn tkinter canvas and make your own star shaped button. Add click bindings to drawn shapes and it will act as a button.
